src/components/accessible-autocomplete/accessible-autocomplete.yaml

```diff
@@ -129,10 +129,10 @@ examples:
         text: Select country
       hint:
         text: "Please select a country"
-      errorMessage: "An error occured"
+      errorMessage: "An error occurred"
       id: location-picker
       select: |
-        <select id="location-picker" data-show-all-values="false" data-auto-select="false" data-default-value="" data-module="hmrc-accessible-autocomplete">
+        <select class="govuk-select govuk-select--error" aria-describedby="location-picker-error location-picker-hint" id="location-picker" data-show-all-values="false" data-auto-select="false" data-default-value="" data-module="hmrc-accessible-autocomplete">
```

Comment:
had to add these, because our markup wasn't actually what a select in error state would look like so VRT will have been wrong for this